packages/cryptoplease/lib/features/activities/src/components/payment_request_tile.dart

```diff
@@ -81,19 +79,30 @@ class _PaymentRequestTileState extends State<PaymentRequestTile> {
             key: ValueKey(widget.id),
             create: (_) => sl<PaymentRequestVerifierBloc>(param1: data),
             lazy: false,
-            child: ListTile(
-              onTap: () =>
-                  context.navigateTo(LinkDetailsFlowRoute(id: data.id)),
-              leading: CircleAvatar(
+            child: ActivityTile(
+              icon: CircleAvatar(
                 radius: 21,
                 backgroundColor: CpColors.yellowColor,
                 child: Assets.icons.incoming.svg(),
               ),
-              title: Text(title(), style: titleStyle),
-              subtitle: Text(
-                context.formatDate(data.created),
-                style: subtitleStyle,
-              ),
+              title: title(),
+              subtitle: context.formatDate(data.created),
+              actions: [
+                CpButton(
+                  text: 'Remind me',
```

Comment:
Should be in l10n. Also, the text is misleading. It should rather be "Open" or smth like this.